docs/tasks/administer-cluster/guaranteed-scheduling-critical-addon-pods.md

```diff
@@ -35,18 +41,33 @@ while the other pods shouldn't tolerate the taint. The taint is removed once the
 
 *Warning:* currently there is no guarantee which node is chosen and which pods are being killed
 in order to schedule critical pods, so if rescheduler is enabled your pods might be occasionally
-killed for this purpose.
+killed for this purpose. Please ensure that rescheduler should not enabled along with priorities & preemptions in default scheduler as it may cause unwanted side-effects.
 
 ## Config
 
-Rescheduler should be enabled by default. It doesn't have any user facing configuration (component config) or API.
+Rescheduler should be enabled by default(unless you enabled priorities & preemption). It doesn't have any user facing configuration (component config) or API.
```

Comment:
This statement is confusing. Why we require/encourage a feature to be enabled even if it is being deprecated?